src/components/searchBar/searchBar.css

```diff
@@ -19,17 +23,55 @@
     transform: rotate(90deg);
 }
 
-.filter-container {
-    position: absolute;
-    top: 2rem;
-    left: 20rem;
-    background: var(--white-color);
-    min-width: 25rem;
-    box-shadow: 0px 0px 2px 1px var(--border-color);
-    border-radius: 8px;
-    z-index: 2;
+.basic-modal {
+    top:3.4rem;
+    background: transparent;
+}
+
+.modal {
+    margin-left: 50%;
 }
 
 .filter-footer {
     border-top: 1px solid var(--border-color);
+}
+
+/* || Media Queries */
+
+@media screen and (max-width: 1080px) {
+    .search-container {
+        width: 20rem;
+    }
+    .modal {
+        margin-left: 40% ;
```

Comment:
Don't use large value for margins instead try to use flex or grid for that

Reply:
Okay. Will fix this in next PR